alphadia/outputtransform.py

```diff
@@ -395,9 +395,7 @@ def build_transfer_model(self):
 
         device = utils.get_torch_device(self.config["general"]["use_gpu"])
 
-        tune_mgr = FinetuneManager(
-            device=device, settings=self.config["transfer_learning"]
-        )
+        tune_mgr = FinetuneManager(device=device, **self.config["transfer_learning"])
```

Comment:
I agree that this pattern defines the defaults twice.

But it decouples it from the config which simplifies use in Notebooks, scripts etc. Which is used a lot for fine-tuning. 
We also use this for other classes like the libtransforms which need to be used from jupyter notebooks.

If we would remote the `**` unpacking, it would decouple config even more but I'm fine like this.

Reply:
after personal discussion:

for now, expand all config values explicitly
(e.g. like in 
```python
        fasta_digest = libtransform.FastaDigest(
            enzyme=self.config["library_prediction"]["enzyme"],
            fixed_modifications=_parse_modifications(
                self.config["library_prediction"]["fixed_modifications"]
            ),
            variable_modifications=_parse_modifications(
                self.config["library_prediction"]["variable_modifications"]
            ),
            max_var_mod_num=self.config["library_prediction"]["max_var_mod_num"],
            missed_cleavages=self.config["library_prediction"]["missed_cleavages"],
            precursor_len=self.config["library_prediction"]["precursor_len"],
            precursor_charge=self.config["library_prediction"]["precursor_charge"],
            precursor_mz=self.config["library_prediction"]["precursor_mz"],
        )
```